components/director/pkg/graphql/schema.graphql

```diff
@@ -658,7 +666,6 @@ type Application {
 	bundle(id: ID!): Bundle
 	auths: [SystemAuth!]
 	eventingConfiguration: ApplicationEventingConfiguration
-	ready: Boolean!
```

Comment:
I see that our `pkg/graphql` entities still have a BaseEntity which contains a `ready` field? Not sure we want to keep it there, but maybe it's fine to stay as long as the end result from the GraphQL query does not contain the field. Have you tried it?

Reply:
Yes, I've tested it and when the query request has the ready field it fails, regardless a base entity has a ready field